File: Source/AngularApplication/App/Models/AuthorizationGroup.ts
"use strict";

module OrganizationRegister
{
    //export class AuthorizationGroup implements Affecto.Base.IModel
    //{
    //    constructor(public name: string, public role: UserRole)
    //    {
    //    }
    //}


    export class AuthorizationGroup implements Affecto.Base.IModel {
        constructor(public name: string, public roleId: string, public roleName: string) {
        }
    }

    
}
 

File: Source/OrganizationRegister.Common/AuthorizationGroup.cs
using System;


namespace OrganizationRegister.Common
{ 

    public class AuthorizationGroup 
    {

        public AuthorizationGroup()
        {
        }

        public AuthorizationGroup(string name, Guid roleId, Guid? groupId)
        {
            Name = name;
            RoleId = roleId;
            GroupId = groupId;
        }

        public string Name { get; set; }
        public Guid RoleId { get; set; }
        public Guid? GroupId { get; set; }

        //PM 20.4.2021
        public Guid? OrganizationId { get; set; }

    }

}